config/application.yml.default

```diff
@@ -398,7 +398,7 @@ development:
   recurring_jobs_disabled_names: "[]"
   s3_report_bucket_prefix: ''
   s3_report_public_bucket_prefix: ''
-  saml_endpoint_configs: '[{"suffix":"2021","secret_key_passphrase":"trust-but-verify"},{"suffix":"2022","secret_key_passphrase":"trust-but-verify"}]'
+  saml_endpoint_configs: '[{"suffix":"2022","secret_key_passphrase":"trust-but-verify"},{"suffix":"2023","secret_key_passphrase":"trust-but-verify"}]'
```

Comment:
We also need to rename these files:
- `config/artifacts.example/local/saml2021.crt`
- `config/artifacts.example/local/saml2021.key.enc`


I we wanted to be really kind to ourselves, we'd regenerate them, I just `openssl -cert`'d the 2022 one and it expired in 2021

(it also looks like we just symlinked 2023 to 2022)

Reply:
Is that not covered in [these edits here](https://github.com/18F/identity-idp/pull/8122/files#diff-25d755ca38bad851ff486f165d753d5c4e2d282a3e6c5188a561e3903ec2ec4cL5-L6)?

I'm also fine with removing the 2022 one(s) and _**just**_ having the 2023 ones everywhere.

Reply:
they work, but I guess we should remove the 2021 files from the repo if they're no longer referenced

Reply:
Huh, I completely missed that we had actual sample cert files there, too. I'll remove and re-test.